components/Button/src/stories/bem.stories.mdx

```diff
@@ -650,3 +650,414 @@ import readme from "../../README.md";
+              fill="currentColor"
+            />
+          </svg>
+        </span>
+      </button>
+      <button
+        disabled
+        class="denhaag-button denhaag-button--end-icon denhaag-button--icon-only denhaag-button--disabled"
+      >
+        <span class="denhaag-button__icon">
+          <svg
+            width="1em"
+            height="1em"
+            viewBox="0 0 20 20"
+            xmlns="http://www.w3.org/2000/svg"
+            class="denhaag-icon"
+            focusable="false"
+            aria-hidden="true"
+          >
+            <path
+              d="M7.74408 15.5899C7.41864 15.2645 7.41864 14.7368 7.74408 14.4114L12.1548 10.0007L7.74408 5.58991C7.41864 5.26447 7.41864 4.73683 7.74408 4.4114C8.06951 4.08596 8.59715 4.08596 8.92259 4.4114L13.9226 9.4114C14.248 9.73683 14.248 10.2645 13.9226 10.5899L8.92259 15.5899C8.59715 15.9153 8.06951 15.9153 7.74408 15.5899Z"
+              fill="currentColor"
+            />
+          </svg>
+        </span>
+      </button>
+    </div>
+  </Story>
+  {/* eslint-enable react/no-unknown-property */}
+</Canvas>
+
+### Icon only - focus
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Icon only - focus">
+    <div class="denhaag-button-group denhaag-button-group--multiple">
+      <button class="denhaag-button denhaag-button--end-icon denhaag-button--icon-only denhaag-button--focus">
+        <span class="denhaag-button__icon">
+          <svg
+            width="1em"
+            height="1em"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+            class="denhaag-icon"
+            focusable="false"
+            aria-hidden="true"
+          >
+            <path d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"></path>
+          </svg>
+        </span>
+      </button>
+      <button class="denhaag-button denhaag-button--secondary-action denhaag-button--end-icon denhaag-button--icon-only denhaag-button--focus">
+        <span class="denhaag-button__icon">
+          <svg
+            width="1em"
+            height="1em"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+            class="denhaag-icon"
+            focusable="false"
+            aria-hidden="true"
+          >
+            <path d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"></path>
+          </svg>
+        </span>
+      </button>
+    </div>
+    <div class="denhaag-button-group denhaag-button-group--multiple">
+      <button class="denhaag-button denhaag-button--end-icon denhaag-button--icon-only denhaag-button--focus">
```

Comment:
This element doesn't have any readable text for screenreaders, therefor the `sr-only` element or `aria-label` attribute must be applied. So the Screenreader mentions what this does.